runner_test.go

```diff
@@ -103,3 +108,89 @@ func TestRunner_appendSecrets(t *testing.T) {
 		})
 	}
 }
+
+func TestRunner_configEnv(t *testing.T) {
+	t.Parallel()
+
+	tt := []struct {
+		name      string
+		env       map[string]string
+		pristine  bool
+		custom    []string
+		whitelist []string
+		blacklist []string
+		output    map[string]string
+	}{
+		// pristine env with no custom vars leads to empty env
```

Comment:
Why have the more verbose comments instead of just putting the desired details in the names?